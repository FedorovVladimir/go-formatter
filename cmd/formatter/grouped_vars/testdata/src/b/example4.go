package main

var g = "g" // want "grouped vars"
var h = "h"

var i = "i" // want "grouped vars"
var j = "j"
