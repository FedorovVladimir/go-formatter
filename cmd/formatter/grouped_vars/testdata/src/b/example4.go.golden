package main

var (
	g = "g" // want "grouped vars"
	h = "h"
)

var (
	i = "i" // want "grouped vars"
	j = "j"
)
